operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/internal/CustomResourceEventSource.java

```diff
@@ -109,38 +111,6 @@ public void eventReceived(Watcher.Action action, T customResource) {
     }
   }
 
-  private void markLastGenerationProcessed(T resource) {
```

Comment:
sorry, not following why this was removed ? I mean the whole concept or generation awareness? Or am I missing something?